packages/theming/README.md

```diff
@@ -0,0 +1,7 @@
+# Theming Across Platforms
+[Apple-theme](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/apple-theme/README.md)  
+[Theme-types](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/theme-types/README.md)  
+[win32-theme](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/win32-theme/README.md)  
+[Theme-tokens](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/theme-tokens/README.md)  
+[Android-theme](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/android-theme/README.md)  
+[default-theme](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/default-theme/README.md)  
```

Comment:
Could we divide this a bit more:

## Default theme if no theme is given:
...
## Platform Themes that implement FluentUI for each platform
...
## Theme types and utilities
... (I would include the theme-utils package here too)